### PR TITLE
some_code created

### PR DESCRIPTION
**Pull request by RSchauer-Home**

### CODE REVIEW

**Comment by RSchauer-Home** [comment]
@danielheger

**Comment by RSchauer-Home** [comment]
@danielheger  Hallo Hallo

File: some_code [hunk 1]
```diff
@@ -0,0 +1 @@
+blabla
```
Comment (danielheger) [inline some_code:1]: Will get the job done!

Comment (danielheger) [reply some_code:1]: Could be improved by 
* Adding the correct file extension
* Buying pizza
* writing actual code

**Review by danielheger — APPROVED** [review]

**Pull request merged**